### DISCUSSION TITLE
Consent request

### DISCUSSION THREAD

**Discussion by derzico** [authelia/authelia#4895] [Q&A] [answered]
Hey there,

I implemented Authelia and I'm using it for a few services with OpenID Connect.
I want to get rid of the consent request after I logged in to authelia.

How can you do that?

![image](https://user-images.githubusercontent.com/75365699/217287283-4ab73e9e-aabb-4689-b85c-e42e4bfda1bf.png)

**Comment by james-d-elliott** [comment, marked as answer]
You can configure the consent mode to implicit or pre-configured for each client (in the [config](https://www.authelia.com/configuration/identity-providers/open-id-connect/)) which you want this behavior to change. Keeping in mind that not showing consent (using the implicit mode) is not technically permitted by the OpenID Connect 1.0 specifications, and is particularly problematic for the public client type.

**Reply by vitSkalicky** [reply]
What are the risks of the 'implicit' consent for 'public: false' clients? I'm in control of the secret (it's saved on my server, not accessible to clients AFAIK), so it should be completely safe.

Side-note: Realistically, the users will soon learn to accept every such dialog if it is too common making it practically same as 'implicit'.
